lib/queue.js

```diff
@@ -551,6 +551,13 @@ Queue.prototype.processStalledJob = function(job){
           if(!isMember){
             return _this.processJob(job);
           }
+
+          return job.releaseLock(_this.token);
+        }).catch(function(err) {
```

Comment:
I think using disposers would be ideal to handle lock resources overall, but that will entail some refactor work to do and I'm not sure if it's worth it to do it on our v0.7.2 fork, maybe as a contribution to 1.0 once we upgrade to 1.0